mkosi

```diff
@@ -1712,6 +1712,12 @@ def install_boot_loader_debian(args, workspace):
     run_workspace_command(args, workspace,
                     "/usr/bin/kernel-install", "add", kernel_version, "/boot/vmlinuz-" + kernel_version)
 
+def install_boot_loader_ubuntu(args, workspace):
```

Comment:
Is there a difference between this function and `install_boot_loader_debian`? They look identical to me (but I haven’t verified it).

Reply:
It is indentical. Dunno, maybe this one should just call the other one?